### PR TITLE
feat(main): fix root and metadata

### PR DESCRIPTION
**Pull request by cuisongliu**

### CODE REVIEW

File: runtime/init.go [hunk 1]
```diff
@@ -106,7 +102,18 @@ func (d *Default) initRunner(cluster *v1.Cluster) error {
 
 	return nil
 }
-
+func (d *Default) CopyMetadata() error {
```
Comment (fanux) [inline runtime/init.go:105]: Why we need copy Metadata?

Comment (cuisongliu) [reply runtime/init.go:105]: at init,applyNodes,applyMasters process need Metadata,  but Metadata in rootfs   and read metadata not mount /tmp/<cluster-name> yet , read metada at runtime init func so far.

Comment (fanux) [reply runtime/init.go:105]: Using /var/lib/sealer/../Metadata

Comment (cuisongliu) [reply runtime/init.go:105]: yes, i using it .but in runtime init this dir not exists  yet

Comment (fanux) [reply runtime/init.go:105]: Newruntime after Mount image, so that you needn't copy metadata

**Review by fanux — APPROVED** [review]

**Pull request merged**